### ISSUE TITLE
Review-2019.3.23-彭晓婷

### ISSUE THREAD

**Issue by lab517-student** [DIYer22/weekly-review#456]
# Review-2019.3.23-彭晓婷

## Preview

- 论文-2018 Annotation-Free and One-Shot Learning for Instance Segmentation of Homogeneous Object Clusters

算法的逻辑和原理基本理解了，但是还没有进行代码实现。待解决的几点：

1. COCO数据集的评估指标我现在还没看懂，打算看看相关资料。
2. 贝叶斯优化中具体的迭代方式文中没写，不知道代码中有没有呈现。
3. 具体的怎么光照共享操作、贝叶斯模拟后验分布文中没再展开，需要另行查阅资料。

算法流程：

1. 获取数据是通过单个视频拍摄单个物体利用KNN、光流算法获取三分图，从而得到大量单目标图像；

2. 结构似然函数的获得：通过CNN（finetuning the AlexNet pretained model）训练，随机生成图像为负样本，真实图像为正样本（30000个）

   CNN参数设置：a learning rate of 0.001, minibatch size of 32, momentum of 0.9 and weight
   decay of 0.01 

3. 合成HOC图像：贝叶斯优化（我的理解是将步骤一获取的单目标图像去一步步迭代合成HOC，但是就之前提到的迭代方式，就是文中提到的旋转因子、大小扩缩因子、移动坐标是预设，但是预设值并没有在文中体现；而且对于贝叶斯优化，它只提到模拟后验分布，但是对于具体操作也是没在文中体现）+光照共享

4. 实例分割用Mask R-CNN（我理解是利用步骤三合成图像2000张训练Mask R-CNN）

- 课程：《数据融合与智能决策》PPT，主要讲了“基于D-S证据理论的数据融合”。
- 安装：下载了相关软件，包括Anaconda、Typra、Mendeley等，但是目前的anaconda还没安装成功，估计是环境变量的关系，但是目前修改后仍旧安装不了，之后再装吧。

## Next

- 解决上面提到的待解决问题
- 了解分割的历史发展历程
- 学习相关计算机视觉的一些概念，防止文献阅读障碍，抽时间整理经典的图像分割方法

**Comment by ilydouble** [comment]
我们组的先修任务已经发给你了。
